rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/dateFilter.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { mount, VueWrapper } from "@vue/test-utils";
+import DateFilter from "../dateFilter.vue";
+import DateTimePicker from "../dateTimePicker.vue";
+import _ from "lodash";
+
+interface MountOptions {
+  props?: Record<string, unknown>;
+  slots?: { default?: string };
+  global?: {
+    stubs?: Record<string, unknown>;
+    mocks?: Record<string, unknown>;
+  };
+}
+const mountDateFilter = (options: MountOptions = {}): VueWrapper<any> => {
+  return mount(DateFilter, {
+    props: {
+      modelValue: { enabled: false, datetime: "" },
+      ...options.props,
+    },
+
+    global: {
+      stubs: {
+        btn: true,
+        DateTimePicker: true,
+        Dropdown: {
+          template: '<div class="dropdown"><slot name="dropdown"></slot></div>',
+        },
```

Comment:
suggestion for this and future tests, it's better to import the dropdown, it will be fewer things to mock/reproduce and then if necessary add data-test attributes in the dropdown, it won't be needed to add in the stub as well.